lighthouse-core/audits/works-offline.js

```diff
@@ -7,6 +7,20 @@
 
 const URL = require('../lib/url-shim.js');
 const Audit = require('./audit.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  title: 'Current page responds with a 200 when offline',
+  failureTitle: 'Current page does not respond with a 200 when offline',
+  description: 'If you\'re building a Progressive Web App, consider using a service worker ' +
+      'so that your app can work offline. ' +
+      '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/http-200-when-offline).',
+  warningNoLoad: 'You may be not loading offline because your test URL ' +
```

Comment:
`You may be not` -> `You may not be`

Reply:
if we're reworking, how about ditching the "You" language entirely?

`The page may not be loading offline because the test URL was redirected...`